test/rubygems/test_project_sanity.rb

```diff
@@ -1,28 +1,9 @@
 # frozen_string_literal: true
 
-require "rubygems/test_case"
 require "open3"
 
 class TestProjectSanity < Minitest::Test
 
-  def test_rake_package_builds_ok
```

Comment:
Isn't this the `more robust test` that you point to in your issue comment?

Reply:
No, please double check the references. The test I'm referring to `test_manifest_is_up_to_date`.

The "more robust test" is not a very accurate expression though, I meant something like the "more focused test" better. When I added `test_rake_package_builds_ok` the idea was to check that the manifest was up to date and `test_manifest_is_up_to_date` does a better job at that.